ssl/statem/extensions_clnt.c

```diff
@@ -893,6 +880,15 @@ EXT_RETURN tls_construct_ctos_key_share(SSL_CONNECTION *s, WPACKET *pkt,
         }
     }
 
+#ifndef OPENSSL_NO_ECH
```

Comment:
this needs one more space of indentation

Reply:
yep